webui/react/src/components/Searches/Searches.tsx

```diff
@@ -0,0 +1,950 @@
+    );
+    const excludedIdSet = new Set(
+      settings.selection.type === 'ALL_EXCEPT' ? settings.selection.exclusions : [],
+    );
+    experiments.forEach((e, index) => {
+      Loadable.forEach(e, ({ experiment }) => {
+        const mapToAdd =
+          (selectAll && !excludedIdSet.has(experiment.id)) || selectedIdSet.has(experiment.id)
+            ? selectedMap
+            : excludedMap;
+        mapToAdd.set(experiment.id, { experiment, index });
+      });
+    });
+    return [excludedMap, selectedMap];
+  }, [isLoadingSettings, selectAll, settings.selection, experiments]);
+
+  const selection = useMemo<GridSelection>(() => {
+    let rows = CompactSelection.empty();
+    if (selectAll) {
+      Loadable.forEach(total, (t) => {
+        rows = rows.add([0, t]);
+      });
+      excludedExperimentIds.forEach((info) => {
+        rows = rows.remove(info.index);
+      });
+    } else {
+      selectedExperimentIds.forEach((info) => {
+        rows = rows.add(info.index);
+      });
+    }
+    return {
+      columns: CompactSelection.empty(),
+      rows,
+    };
+  }, [selectAll, selectedExperimentIds, excludedExperimentIds, total]);
+
+  const colorMap = useGlasbey([...selectedExperimentIds.keys()]);
+
+  const experimentFilters = useMemo(() => {
+    const filters: V1BulkExperimentFilters = {
+      projectId: project.id,
+    };
+    return filters;
+  }, [project.id]);
+
+  const numFilters = useMemo(() => {
+    return (
+      Object.values(experimentFilters).filter((x) => x !== undefined).length -
+      1 +
+      rootFilterChildren.length
+    );
+  }, [experimentFilters, rootFilterChildren.length]);
+
+  const handleSortChange = useCallback(
+    (sorts: Sort[]) => {
+      setSorts(sorts);
+      const newSortString = makeSortString(sorts.filter(validSort.is));
+      if (newSortString !== sortString) {
+        resetPagination();
+      }
+      updateSettings({ sortString: newSortString });
+    },
+    [resetPagination, sortString, updateSettings],
+  );
+
+  useEffect(() => {
+    if (!isLoadingSettings && settings.sortString) {
+      setSorts(parseSortString(settings.sortString));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isLoadingSettings]);
+
+  const fetchExperiments = useCallback(async (): Promise<void> => {
+    if (isLoadingSettings || Loadable.isNotLoaded(loadableFormset)) return;
+    try {
+      const tableOffset = Math.max((page - 0.5) * PAGE_SIZE, 0);
+
+      // always filter out single trial experiments
+      const filters = JSON.parse(filtersString);
+      filters.filterGroup.children.push({
+        children: [
+          {
+            columnName: 'numTrials',
+            kind: 'field',
+            location: 'LOCATION_TYPE_EXPERIMENT',
+            operator: '>',
+            type: 'COLUMN_TYPE_NUMBER',
+            value: 1,
+          },
+        ],
+        conjunction: 'and',
+        kind: 'group',
+      });
```

Comment:
this won't work as intended if the top level filter group has a `or` conjunction -- you have to wrap the existing filter group with a group that has your experiment clause AND the existing filter group.